Form/Type/CollectionType.php

```diff
@@ -31,6 +34,30 @@ public function getParent()
     }
 
     /**
+     * {@inheritdoc}
+     */
+    public function buildView(FormView $view, FormInterface $form, array $options)
```

Comment:
Please move `buildView` and `setDefaultOptions` on top.

`getParent` should be placed after them.